lib/new_relic/telemetry_sdk/harvester.rb

```diff
@@ -52,7 +52,7 @@ def stop
 
       def harvest
         @lock.synchronize do
-          @harvestables.each do |harvestable|
+          @harvestables.each do |_name, harvestable|
```

Comment:
Suggestion from mobbing: change to `@harvestables.values.each` to avoid dealing with `name` entirely.